src/pages/NotFound.spec.js

```diff
@@ -2,7 +2,7 @@ import { MemoryRouter } from 'react-router-dom';
 import { render, screen } from '@testing-library/react';
 import NotFound from './NotFound';
 
-describe('Navigation', () => {
+describe('NotFound', () => {
```

Comment:
It is not clear whether `NotFound` is a page or a component; I would suggest to make that clear here in order to make sure that all fellow developers understand what the test it doing.